Softeq.XToolkit.Common.Droid.Tests/Extensions/EditTextExtensionsTests/InputFilterMock.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Android.Text;
+using Java.Lang;
+
+namespace Softeq.XToolkit.Common.Droid.Tests.Extensions.EditTextExtensionsTests
+{
+    public class InputFilterMock : Object, IInputFilter
```

Comment:
Could you please rename to `MockInputFilter` for consistency?

Reply:
Sure, done)